app/assets/stylesheets/views/_homepage.scss

```diff
@@ -36,6 +36,7 @@ body.homepage {
 }
 
 .home-top__intro {
+  @include govuk-font(19);
```

Comment:
This change has reduced the space on mobile between this text and the 'Simpler, clearer, faster' text beneath it. Was this intentional? It looks a little squashed, maybe worth adding a bit of margin?

Reply:
It seems like we used to have a leaking 1.5 on line-height. Will update to bring it back to the text. Good eye!

Reply:
Actually, no. It was [originally 1.25](http://web.archive.org/web/20210131010240/https://www.gov.uk/), so I wouldn't add custom line-height to it.